pkg/metrics/tool/common.go

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package tool
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func FormatFloat64(f float64, point int) float64 {
+	var value float64
+	switch point {
+	case 6:
+		value, _ = strconv.ParseFloat(fmt.Sprintf("%.6f", f), 64)
+	case 2:
+		fallthrough
+	default:
+		value, _ = strconv.ParseFloat(fmt.Sprintf("%.2f", f), 64)
+	}
+
+	return value
+}
+
+func NewMetricsTimer(f func(float64)) *prometheus.Timer {
+	return prometheus.NewTimer(prometheus.ObserverFunc(f))
+}
+
+func CollectElapsedTimeWithBeginLabel(g *prometheus.GaugeVec) *prometheus.Timer {
```

Comment:
Can we unify the timers measuring Prepare/Mounts/Remove/Cleanup by using this helper function?

Reply:
Yes, Done.